src/RLP_TMR2023/prueba_behaviour_trees/can_distance.py

```diff
@@ -0,0 +1,94 @@
+import enum
+import logging
+
+import py_trees.common
+import py_trees.console
+
+logger = logging.getLogger(__name__)
+
+class CanViewingToBB(py_trees.behaviour.Behaviour):
+    def __init__(self):
+        super().__init__(name="Can distance to BB")
+        #self.blackboard = self.attach_blackboard_client(name=self.name)
+        #self.blackboard.register_key("distance", access=py_trees.common.Access.WRITE)
+        logger.info("Can distance to BB")
+        return py_trees.common.Status.SUCCESS
+
+class Distances(enum.Enum):
```

Comment:
I think this enum is good, but it should be defined in another place, mark a TODO comment and is ok